bitnami/nginx/templates/health-ingress.yaml

```diff
@@ -3,7 +3,7 @@ Copyright VMware, Inc.
 SPDX-License-Identifier: APACHE-2.0
 */}}
 
-{{- if .Values.healthIngress.enabled }}
+{{- if and .Values.healthIngress.enabled .Values.containerPorts.http }}
```

Comment:
I could accept this change, but I think the ingress backend can be easily changed to point to the https service. Could you take a look?

Reply:
I've done the same `ternary` for the port in the ingress backends.